calm/dsl/builtins/models/tasks.py

```diff
@@ -0,0 +1,18 @@
+from .entity import EntityType
+
+
+# Task
```

Comment:
Any need of this file ?

Reply:
If yes.
These are same task that are defined in `calm/dsl/builtins/models/task.py`.
Try to add your logic somewhat there.

Reply:
removed the file as its not required